explainer-client/src/main/scala/CAPIConfig.scala

```diff
@@ -0,0 +1,8 @@
+
+case class TagObject(id: String, webTitle: String)
+case class CAPIResponseTagsSearchResponseObject(results: List[TagObject])
+case class CAPIResponseTagsSearch(response: CAPIResponseTagsSearchResponseObject)
+
+object CAPIConfig {
+  val capiTagQueryURL = "https://content.guardianapis.com/tags"

```

Comment:
I can see you did this to avoid duplication of the hard-coded url.It's probably best not to introduce another config location - either use the existing one like you're doing with the capi key, or just hard code this in the function (which there should only be one of now)